crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -32,6 +32,73 @@ def role_constraints
     end
   end
 
+  def shutdown_openstack_at_nodes
```

Comment:
I know I am being a pain, but I guess the name is now somewhat misleading. Because it only handles non-database nodes. Not very nice but more precise: ```shutdown_services_at_non_db_nodes```. Feel free to use something else.